beta/src/components/Search.tsx

```diff
@@ -51,6 +51,12 @@ export const Search: React.FC<SearchProps> = ({
 }) => {
   const [isLoaded] = React.useState(true);
   const [isShowing, setIsShowing] = React.useState(false);
+  const actionKey = React.useMemo<string | null>(() => {
+    if (typeof navigator !== 'undefined') {
+      return /(Mac|iPad)/i.test(navigator.platform) ? '⌘' : 'Ctrl';
+    }
+    return null;
+  }, []);
```

Comment:
This seems like it would create a mismatch between server and client, wouldn't it?

I'd suggest either using an effect to set it after first render, or to have an inline script in head that sets a CSS class, and then toggle based on CSS.

Reply:
Yes, but why is that a bad thing in this case? Do you prefer to set the default action key for a prerendered page?
useEffect will cause a re-render every time after a route changed...